### ISSUE TITLE
Prefer GtkFileChooserNative over GtkFileChooser

### ISSUE THREAD

**Issue by JoostvanPinxten** [kees-jan/scroom#85]
Especially on Windows, the Gtk File Chooser looks very different, and does not have all the functionality that I expect to have.

Therefore, I would appreciate (a setting) to use the [GtkFileChooserNative,](https://developer.gnome.org/gtk4/stable/GtkFileChooserNative.html#GtkFileChooserNative-struct) class instead which will fall back to the current one if no native file chooser is available.

**Comment by kees-jan** [comment]
Sounds good. Would it be possible for you to do a pull-request? ;-)

**Comment by JoostvanPinxten** [comment]
Yes, I have that intention. However, I am quite stretched in time. Will do this once the SepGroup8 delivers their changes, as to not conflict with those.
